lib/moonshot/build_mechanism/github_release.rb

```diff
@@ -170,6 +169,27 @@ def fetch_changes(version)
       end
     end
 
+    # Checks for the commit's CI job status. If its not finished yet,
+    # wait till timeout.
+    #
+    # @param sha [String] Commit sha.
+    #
+    # @return [String] Status and links to the CI jobs
+    def check_ci_status(sha)
+      out = nil
+      ilog.start_threaded('Validate `CHANGELOG.md`.') do |step|
+        Retriable.retriable on: RuntimeError,
```

Comment:
Let's use a method to wrap these:

```ruby
def with_retries(&blk)
  default_options = {
    on: [ RuntimeError ],
    multiplier: 1.25
  }
  Retriable.retriable(default_options, &blk)
end
```

By the way, why are we retrying on `RuntimeError` instead of `StandardError`? Are we trying to allow a certain exception to pass through?